.dockerignore

```diff
@@ -0,0 +1,143 @@
+.git
+Dockerfile
+.DS_Store
+.vscode
+.gitignore
+.dockerignore
+
+/credentials
+/cache
+/store
+
+/node_modules
```

Comment:
As the dockerimage is for the backend only, should we ignore `/visualizer` in .dockerignore?

Reply:
Good call, its been added.